client/src/components/transformation/merge/SourceMergeOptions.jsx

```diff
@@ -3,15 +3,19 @@ import PropTypes from 'prop-types';
 import Immutable from 'immutable';
 import { connect } from 'react-redux';
 import { intlShape } from 'react-intl';
+import _ from 'lodash';
 import { ensureDatasetFullyLoaded } from '../../../actions/dataset';
 import SelectDataset from './SelectDataset';
 import SelectMenu from '../../common/SelectMenu';
 import { guessMergeColumn, getColumnName, directionLabels } from './utils';
 import './SourceMergeOptions.scss';
 import { columnName, findColumnI, columnSelectOptions, columnSelectSelectedOption } from '../../../utilities/column';
 
+function capitalizeFirstLetter(string) {
```

Comment:
Since we have lodash as dependency, we can use [capitalize](https://lodash.com/docs/4.17.15#capitalize) ?